crates/turbo-tasks-macros/src/value_trait_macro.rs

```diff
@@ -233,8 +243,6 @@ pub fn value_trait(args: TokenStream, input: TokenStream) -> TokenStream {
                 let raw_vc: turbo_tasks::RawVc = super_trait_vc.into();
                 #ref_ident { node: raw_vc }
             }
-
-            #(pub #trait_fns)*
```

Comment:
Could probably add this back to reduce the imports, though this is more "rusty" now